drawBot/drawBotDrawingTools.py

```diff
@@ -1,39 +1,49 @@
-from typing import Any
-import AppKit # type: ignore
-import CoreText # type: ignore
-import Quartz # type: ignore
 import math
 import os
 import random
 from collections import namedtuple
 from contextlib import contextmanager
+from typing import Any
 
-from .context import getContextForFileExt, getFileExtensions, getContextOptionsDocs
-from .context.baseContext import (
-    makeTextBoxes,
-    getNSFontFromNameOrPath,
-    getFontName,
-)
-from .context.baseContext import FormattedString, BezierPath
-from .context.dummyContext import DummyContext
+import AppKit  # type: ignore # type: ignore
```

Comment:
Not sure why this PR introduces these "type: ignore" comments in the first place, but this one is a bit much.

Reply:
maybe the import sorting of ruff caused this...